python/lib/bidsreader.py

```diff
@@ -90,20 +88,23 @@ def load_bids_data(self):
         if self.verbose:
             print('Loading the BIDS dataset with BIDS layout library...\n')
 
-        bids_config = os.environ['LORIS_MRI'] + "/python/lib/bids.json"
-        exclude_arr = ['/code/', '/sourcedata/', '/log/', '.git/']
+        bids_config   = os.environ['LORIS_MRI'] + "/python/lib/bids.json"
+        exclude_arr   = ['/code/', '/sourcedata/', '/log/', '.git/']
+        force_arr     = [re.compile("_annotations\.(tsv|json)$")]
 
         # BIDSLayoutIndexer is required for PyBIDS >= 0.12.1
         bids_pack_version = list(map(int, bids.__version__.split('.')))
-        if (bids_pack_version[0] > 0
-            or bids_pack_version[1] > 12
-            or (bids_pack_version[1] == 12 and bids_pack_version[2] > 0)):
-            bids_layout = BIDSLayout(
-                root=self.bids_dir,
-                indexer=BIDSLayoutIndexer(config_filename=bids_config, ignore=exclude_arr)
-            )
-        else:
-            bids_layout = BIDSLayout(root=self.bids_dir, config=bids_config, ignore=exclude_arr)
+        # disabled until is a workaround for https://github.com/bids-standard/pybids/issues/760 is found
+        # [file] bids_import.py [function] read_and_insert_bids [line] for modality in row['modalities']: (row['modalities'] is empty)
+        #if (bids_pack_version[0] > 0
+        #    or bids_pack_version[1] > 12
+        #    or (bids_pack_version[1] == 12 and bids_pack_version[2] > 0)):
+        #    bids_layout = BIDSLayout(
+        #        root=self.bids_dir,
+        #        indexer=BIDSLayoutIndexer(config_filename=bids_config, ignore=exclude_arr, force_index=force_arr)
+        #    )
+        #else:
+        bids_layout = BIDSLayout(root=self.bids_dir, config=bids_config, ignore=exclude_arr, force_index=force_arr, derivatives=True)
```

Comment:
This does not work for datasets that have no derivatives. I get the following warnings trying to load the Face13 dataset with no derivatives:
```
/Users/cmadjar/Development/python_venv/python_intel_3.9.4/loris-mri-venv/lib/python3.9/site-packages/bids/layout/validation.py:131: UserWarning: Derivative indexing was requested, but no valid datasets were found in the specified locations (['/Users/cmadjar/Data/LORIS_imaging_data/incoming/EEG/Face13_small/derivatives']). Note that all BIDS-Derivatives datasets must meet all the requirements for BIDS-Raw datasets (a common problem is to fail to include a 'dataset_description.json' file in derivatives datasets).
Example contents of 'dataset_description.json':
{"Name": "Example dataset", "BIDSVersion": "1.0.2", "PipelineDescription": {"Name": "Example pipeline"}}
  warnings.warn("Derivative indexing was requested, but no valid "
```

Reply:
Weird, I think I tested the same case without any problem. Let me double-check.

Reply:
Ok, I see, it's a warning, not an error. Does it prevent the pipeline to run?

Reply:
All good, we can keep the warning. I initially thought that was why the pipeline failed when I tested it but it was because of the task event insertion.